connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "getRowByIndex"
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter the index of the row to add",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must contains only numbers."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "createRow"
+      }
+    },
+    {
+      "label": "Column index",
+      "description": "Enter the index of the column",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "[A-Z]+",
+          "message": "Must contains only capital letters."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "deleteColumn"
+      }
+    },
+    {
+      "label": "Start index",
+      "description": "Enter start index (leave empty if add to the end of the sheet)",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.startIndex"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      }
+    },
+    {
+      "label": "End index",
+      "description": "Enter End index (leave empty if add to the end of the sheet)",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.endIndex"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      }
+    },
+    {
+      "label": "Cell ID",
+      "description": "Enter the ID of the cell ",
```

Comment:
can you add here an example? This makes it easier for the enduser

Reply:
Honestly, I've thought to do it while implementing this operation, as I will able to see how it looks better from user side

Reply:
fine for me, just close this comment as soon as you have a solution :-)

Reply:
For now, I close it, but I'll keep in mind this suggestion